kura/org.eclipse.kura.driver.ble.sensortag.provider/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,39 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BLE SensorTag Driver
+Bundle-SymbolicName: org.eclipse.kura.driver.ble.sensortag.provider;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-License: Eclipse Public License v1.0
+Bundle-Category: Asset-Driver Management
+Bundle-ActivationPolicy: lazy
+Import-Package: javax.security.cert,
+ javax.xml,
+ javax.xml.stream,
+ org.eclipse.kura;version="[1.2,2.0)",
+ org.eclipse.kura.bluetooth.le;version="[1.0,2.0)",
+ org.eclipse.kura.channel;version="1.0.0",
+ org.eclipse.kura.channel.listener;version="[1.0,1.1)",
+ org.eclipse.kura.configuration;version="[1.1,1.2)",
```

Comment:
I believe that even if it is a configurable component we shouldn't import strictly the configuration API. I believe there was a document by @cdealti on that. But I'm not sure.

Reply:
I think so.